project_search.py

```diff
@@ -104,29 +136,45 @@ def check_organization(fastqs, org_paths):
     return not_organized
 
 def main():
-    parser = argparse.ArgumentParser(description="Find runfolders containing a specific project ID.")
-    parser.add_argument("--project", required=True, help="The project ID to search for.")
-    parser.add_argument("--check_org", required=False, action="store_true", help="Check if organization(s) in /proj/ngi2016001/nobackup/DATA/<project>* corresponds to identified samples and runfolders")
-    parser.add_argument("--list_files", required=False, help="List all fastq files that are not organized in specified folder. Requires --check_org")
+    parser = argparse.ArgumentParser(
+        description="Find runfolders containing a specific project ID.")
+    parser.add_argument(
+        "--project", required=True, help="The project ID to search for.")
+    parser.add_argument(
+        "--check_org",
+        required=False,
+        help="Check if organizations in /proj/ngi2016001/nobackup/DATA/ "
+        "correspond to identified samples and runfolders.\n "
+        "Specify an organized folder (e.g., <project>1234) or 'all' to check <project>*",)
```

Comment:
I think it would be useful to set `default="all"`, I guess that is what you usually want? If not it is a good way to get all available options.